### DISCUSSION TITLE
TTGO T-Display does not work after library update

### DISCUSSION THREAD

**Discussion by SalihHortum** [Bodmer/TFT_eSPI#2750] [Alerts]
I am using ARDUINO-IDE , TFT_eSPI library for TTGO T-Display.
The working library version is: Ver:2.2.20. When we update it into newer version display is getting dark. Do not work.
Is there any solution???

**Comment by lewisxhe** [comment]
https://github.com/Bodmer/TFT_eSPI/blob/master/User_Setup_Select.h#L58-L58

If this needs to be manually rewritten as the default configuration for T-Display after the update
